### ISSUE TITLE
Bug: Beta ignores `--no-fail-fast`

### ISSUE THREAD

**Issue by konstin** [nextest-rs/nextest#1955]
### Description of the issue

I'm running my tests with `cargo nextest run --failure-output immediate-final --no-fail-fast`.

### Expected outcome

Previously, the tests would always run to the end.

### Actual result

With 0.9.86-b.1, an error terminates testing early.

### Nextest version

```text
cargo-nextest 0.9.86-b.1 (30f6e4c8a 2024-12-05)
release: 0.9.86-b.1
commit-hash: 30f6e4c8a22e5a35c453ddc4dd543fe8e3a0e4fb
commit-date: 2024-12-05
host: x86_64-unknown-linux-gnu
```

**Comment by sunshowers** [comment]
Thank you so much for beta testing, and for the report! #1956 should address this (and also adds test coverage for part of it)

**Comment by sunshowers** [comment]
Just released 0.9.86-b.2 with a fix for the issue. Would love it if you gave it a go! Thanks again.
